src/main/resources/openapi/courthouse.yaml

```diff
@@ -40,7 +40,31 @@ paths:
                 $ref: '#/components/schemas/CourtList'
         '500':
           description: Internal Server Error
+  /admin/regions:
+    get:
+      tags:
+        - Courthouses
+      summary: Gets all regions
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AdminRegionList'
+        '404':
+          description: Not Found
+          content:
+            application/json+problem:
+              schema:
+                $ref: './problem.yaml'
```

Comment:
We don't have any ACs around 404, is this needed?

Reply:
Removed this from api and therefore the try/catch/throw 404 from the controller